src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -271,6 +275,19 @@ private static boolean shouldBeAloneOnLineWithAloneOrSinglelineOption(Details de
     }
 
     /**
+     * Whether last right curly should be alone on line.
+     * @param details details for validation.
+     * @return true, if last right curly should be alone on line.
+     */
+    private static boolean shouldLastRcurlyBeAloneOnLine(Details details) {
+        final boolean isTokenLambda = details.lcurly.getParent().getType() == TokenTypes.LAMBDA;
+        return !isTokenLambda
+                    && details.rcurly.getLineNo() == details.nextToken.getLineNo()
+                || isTokenLambda
+                    && getNextToken(details.nextToken).getLineNo() == details.rcurly.getLineNo();
```

Comment:
Why aren't these last 2 lines the same as `isAloneOnLineLambda` implementation?
Only difference is one uses `==` and the other uses `!=`.
Both methods talk about being alone on line. Shouldn't this one call the other?